src/tests/ftest/pool/rebuild_with_io.py

```diff
@@ -50,8 +50,7 @@ def test_rebuild_with_io(self):
         :avocado: tags=all,pool,rebuild,pr,medium,rebuildwithio
         """
         # Get the test params
-        pool = TestPool(self.context, self.log,
-                        dmg_command=self.get_dmg_command())
+        pool = TestPool(self.context, self.get_dmg_command(), log=self.log)
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
        pool = TestPool(self.context, self.get_dmg_command())
```